### PR TITLE
Use com.saucelabs as package name

### PR DESCRIPTION
**Pull request by ndmanvar**

### CODE REVIEW

**Comment by johnroach** [comment]
Thank you Neil. Will be working with @tylerdave to find out why our build/test failed.

File: src/main/resources/META-INF/gradle-plugins/io.ndmanvar.SauceConnectPlugin.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+implementation-class=com.saucelabs.SauceConnectPlugin
```
Comment (johnroach) [inline src/main/resources/META-INF/gradle-plugins/io.ndmanvar.SauceConnectPlugin.properties]: I think you need to change the file name here.

File: src/main/resources/META-INF/gradle-plugins/io.johnroach.SauceConnectPlugin.properties [hunk 2]
```diff
@@ -1 +1 @@
-implementation-class=io.johnroach.SauceConnectPlugin
+implementation-class=com.saucelabs.SauceConnectPlugin
```
Comment (johnroach) [inline src/main/resources/META-INF/gradle-plugins/io.johnroach.SauceConnectPlugin.properties]: I believe this can be deleted.

**Review by johnroach — CHANGES_REQUESTED** [review]

**Comment by tylerdave** [comment]
+1

**Pull request merged**